pytest/tests/mocknet/mirror.py

```diff
@@ -235,6 +236,25 @@ def _apply_config_changes(node, state_sync_location):
         do_update_config(node, f'{key}={json.dumps(change)}')
 
 
+def _clear_state_parts_if_exists(args, nodes):
```

Comment:
Out of curiosity what is the naming convention here for functions that start with `_` vs those that don't?

Reply:
Usually refers to a "private" function although python doesn't directly have a concept of private functions.